Aktas2012/Fig5.csv

```diff
@@ -0,0 +1,107 @@
+American Housing Survey Year,Mean Residential Building Lifetime
+1996.998166,51.63861386
```

Comment:
From the image it seems like the authors used discrete values for the individual years. Therefore, I suggest to also use only one datapoint per year. Also, I suggest to use an integer value for the year.